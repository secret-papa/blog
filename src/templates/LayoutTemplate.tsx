import React, { ReactNode } from "react";
// import { Global, ThemeProvider, css } from "@emotion/react";

// import { Header, Footer } from "../components";
// import { resetStyle, theme } from "../styles";

// const useStyles = () => ({
//   root: ({ colors: { grey } }) => css`
//     display: flex;
//     flex-direction: column;
//     background: ${grey[100]};
//     min-height: 100vh;
//   `,
//   content: css`
//     margin: 0 auto;
//     width: 110rem;
//   `,
// });

type LayoutTemplateProps = {
  children?: ReactNode | undefined;
};

const LayoutTemplate = ({ children }: LayoutTemplateProps) => (
  // const styles = useStyles();

  <div>{children}</div>
  // <ThemeProvider theme={theme}>
  //   <Global styles={resetStyle} />
  //   <div css={styles.root}>
  //     <Header />
  //     <div css={styles.content}>{children}</div>
  //     <Footer />
  //   </div>
  // </ThemeProvider>
);
export default LayoutTemplate;
