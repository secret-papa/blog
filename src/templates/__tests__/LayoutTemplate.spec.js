import React from "react";
import { render } from "@testing-library/react";

import LayoutTemplate from "../LayoutTemplate";

describe("<LayoutTemplate />", () => {
  it("should show correctly", () => {
    const { container } = render(<LayoutTemplate />);
    expect(container).toMatchSnapshot();
  });
});
