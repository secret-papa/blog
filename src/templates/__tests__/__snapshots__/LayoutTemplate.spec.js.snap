// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutTemplate /> should show correctly 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background: #F5F5F5;
  min-height: 100vh;
}

.emotion-1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: left;
  -ms-flex-pack: left;
  -webkit-justify-content: left;
  justify-content: left;
  height: 7.2rem;
}

.emotion-2 {
  margin: 0 auto;
  width: 110rem;
}

.emotion-4 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.emotion-6 {
  color: #757575;
  font: bold 3.2rem/1 sans-serif;
  font-family: inherit;
}

<div>
  <div
    class="emotion-0"
  >
    <header
      class="emotion-1"
    >
      <div
        class="emotion-2"
      >
        <div
          class="emotion-3"
        >
          <a
            class="emotion-4 emotion-5"
            href="/"
          >
            <h1
              class=" emotion-6"
            >
              :D
            </h1>
          </a>
        </div>
      </div>
    </header>
    <div
      class="emotion-2"
    />
  </div>
</div>
`;
